apps/console/src/features/core/hooks/use-request.ts

```diff
@@ -121,7 +122,13 @@ export default function useRequest<Data = unknown, Error = unknown>(
                 headers: {},
                 status: 200,
                 statusText: "InitialData"
-            }
+            },
+            shouldRetryOnError: (error) => {
```

Comment:
Lets do this in the request level.

```diff
    const requestConfig: RequestConfigInterface = {
        headers: {
            "Accept": "application/json",
            "Content-Type": "application/json"
        },
        method: HttpMethods.GET,
        params: {},
        url: store.getState().config.endpoints.myAccountConfigMgt + "/status/enable"
    };

    const { data, error, isValidating, mutate } = useRequest<Data, Error>(requestConfig, {
+        shouldRetryOnError: false
    });

    return {
        data,
        error: error,
        isLoading: !data && !error,
        isValidating,
        mutate
    };
```

Ref: https://swr.vercel.app/docs/options#options